<launch>
	<group ns="sim">
		<param name="tf_prefix" value="sim" />
		<rosparam command="load" ns="move_group" file="$(dirname)/../config/controllers_sim.yaml" />
		<!-- Launch moveit_planning_execution.launch from motoman_gp4_moveit_config package under
		the 'yk_architect' namespace -->
		<include file="$(find motoman_gp4_moveit_config)/launch/moveit_planning_execution.launch">
			<arg name="sim" value="true" />
			<arg name="default_action_ns" value="false" />
		</include>

		<!-- Start move_for_lego node -->
		<node name="yk_interface" pkg="yk_interface" type="yk_interface" output="screen">
			<param name="group_name" type="str" value="manipulator" />
		</node>

	</group>
</launch>